docs/CodeReviewRulesOfThumb.md

```diff
@@ -0,0 +1,76 @@
+# Rules of Thumb for Code Reviews
+
+This document aims to outline a set of best practices that are followed, within reason, when reviewing PRs.
+
+## Select a reviewer
+
+Author:
+- Manually assign at least one reviewer when creating a PR
+- Let the reviewer know that you have selected them during scrum (or ping them over Teams)
+
+Reviewer:
+- Indicate right away if you are the wrong reviewer (you won't be able to review the PR if you are going on vacation, for example)
```

Comment:
Thoughts on rephrasing slightly?

```suggestion
- Indicate right away if you won't be able to or don't feel comfortable to sign-off as the reviewer (i.e. you won't be able to review the PR because you are going on vacation)
```

Reply:
+1 good idea. Fixed in 47d06c7.